src/App.jsx

```diff
@@ -1,13 +1,44 @@
 import AdviceSection from './sections/Advice'
 import ArtsSection from './sections/Art'
 import UsersSection from './sections/Users'
+
 import './App.css'
+import { useState } from 'react'
+import { useEffect } from 'react'
 
 function App() {
+  const [artdata, setArtData] = useState([]);
+  const [userData, setUserData] = useState([]);
+
+  const GET_ART = "https://boolean-api-server.fly.dev/art";
```

Comment:
these can be defined outside the function

Reply:
Yes, Done!